exercises/farewell-workflow/README.md

```diff
@@ -0,0 +1,59 @@
+# Temporal 101 Farewell Exercise
+
+# Exercise 3: Farewell Workflow
+During this exercise, you will create an Activity function,
+register it with the Worker, and modify a Workflow to execute it.
+
+Before continuing, press Ctrl-C to stop any Worker instances still running from any
+previous exercises.
+
+As with other exercises, you should make your changes in the `practice` 
+directory. Look for "TODO" comments, which will provide hints about what
+you'll need to change. If you get stuck and need additional hints, or 
+if you want to check your work, look at the completed example in the
+`solution` directory. 
+
+## Part A: Write an Activity Function
+The `TranslateActivities.cs` file contains a method(`GetSpanishGreeting`) that uses a 
+microservice to get a customized greeting in Spanish. Follow these steps to create 
+a new Activity that gets a custom farewell message from the microservice:
+
+1. Open the `TranslateActivities.cs` file. This is located in the `practice/Workflow`
+   subdirectory under the `farewell-workflow` directory in `exercises`.
+2. Create a new Activity that will get a custom farewell message.
+   1. Copy the `GetSpanishGreeting` method to create a new Activity.
+   2. Rename the new `GetSpanishGreeting` function to `GetSpanishFarewell`.
```

Comment:
Add a line here instructing the learner to modify the Activity to call the appropriate URL

```
1. Modify the Activity to call `get-spanish-farewell`
```